Gems/ROS2/Code/Source/RobotImporter/Utils/RobotImporterUtils.h

```diff
@@ -66,6 +66,7 @@ namespace ROS2
         AZStd::string ResolveURDFPath(
             AZStd::string unresolvedPath,
             const AZStd::string& urdfFilePath,
+            const AZStd::string& amentPrefixPath,
```

Comment:
update: It would be great to update the API comment above to include the new `amentPrefixPath` parameter